packages/bentley/grouping-mapping-widget/src/api/generated/git_push.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
what is this doing here?

Reply:
Left over from the generated client. Will remove.

Reply:
This whole client is temporary as we are planning on building our own reporting client.